docs/extensions/tutorials/_posts/1970-01-08-ModifiyingExtensions.md

```diff
@@ -125,6 +125,18 @@ This is what we wanted! It was quite easy and straightforward. However, doing th
 
 To get those updates from Shoutem, we want to extend the extension instead of changing its code directly.
 
+### Debugging directly edited extensions
+
+When directly editing Shoutem extensions you will have to either re-clone the app after uploading and installing the extension into your app or re-naming the extension's directory name locally. This will reveal any underlying bugs with `import`s.
+
```

Comment:
In my opinion, there are no "underlying" bugs with imports. The imports are working correctly. I'd focus on what happens when you publish an extension that's not yours - "Fork" and what happens. If you describe it correctly it's understandable that some imports are then obsolete. Also, explain that what happens is this:
1. on server side: a new extension is created, and published under your username: devName.extensionName.
Now on a server side, your app has original extension installed, and new (your) extension installed
2. on client side: installation is swapped, original extension is now swapped with your extension, now if someone else is trying to load/import from original extension, this won't work anymore. If you encounter such behaviour, it's best to re-clone the app, so that both original extension and your fork are cloned.

Reply:
Regarding 2.:
It will work, the issue is when the original extension is uninstalled. I'll reformulate it and request review again.